types/stripe/index.d.ts

```diff
@@ -36,88 +36,95 @@
 
 /// <reference types="node" />
 
+import { Agent } from 'http';
+
 declare class Stripe {
-    DEFAULT_HOST: string;
```

Comment:
Could you avoid unrelated formatting changes here? This file uses 4 spaces.

Reply:
On another note, I think this needs an explicit Prettier config.

Reply:
I did what the docs said and ran `npm run lint package-name` - this caused the formatting changes?

Reply:
It hasn't for me in the past - most likely your editor has auto-format on save enabled?